src/pages/Project/pages/About.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Header from '../components/Header';
+
+function About(props) {
+  console.log(props);
```

Comment:
Is this here for debug purposes?

Reply:
Oh, whoops! Yes, for debugging.

Reply:
I have other changes suggested by Alex, so feel free to request changes for this PR.